.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "projects/tuist/fixtures/tuist_plugin"]
+	path = projects/tuist/fixtures/tuist_plugin
+	url = https://github.com/fortmarek/ExampleTuistPlugin/
```

Comment:
Are we sure about adding a submodule, can't the test just check out the plugin? Also, should we put it inside the `tuist` repository?

Reply:
it still has to be its own separate repo for acceptance tests where it's used as a plugin in a `Config.swift`. But it is also used as a plugin itself which is where I wanted to reuse git submodule to have the code locally in the repo. I'd rather have it inside the repo to start with but I can't make it work for the `Config.swift` scenario.